models.py

```diff
@@ -18,33 +20,31 @@ def mostrecent(x):
     return preds
 
 def arima(X):
-    """Predict the most recent value using ARIMA(2,1,0) model.
+    """Predict the most recent value using ARIMA(1,1,1) model.
```

Comment:
You and Alec have both made changes to the ARIMA parameters (in separate PRs) that improve the performance. As written, these changes will conflict with one another. How about we make the ARIMA parameters an optional argument? So the interface will be `arima(X, order=(1,1,1))`?

Reply:
Yup!

Reply:
Done.